incubator/hnc/internal/validators/object_test.go

```diff
@@ -273,8 +281,24 @@ func TestUserChanges(t *testing.T) {
 			},
 		},
 	}, {
-		name: "Deny deletions of propagated objects",
-		fail: true,
+		name:       "Deny deletions of propagated objects when namespace is not being deleted",
+		fail:       true,
+		isDeleting: false,
```

Comment:
`false` is the zero-value of `boolean` so I'd prefer to leave the out here.

Reply:
afaik, is explicit better than implicit?

Reply:
In most of our tests, we leave out the zero values, because otherwise every time you add a new flag for just one test, you have to go update all the other tests. This ends up making the tests harder to read IMO instead of just emphasizing the parts of the test that are important.

We tend not to rely on zero values as much in the real code, but tests are a bit of a different story.